ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,21 @@
-﻿namespace ConsoleApp1
+﻿using System.Diagnostics;
+
+namespace ConsoleApp1
 {
 	internal class Program
 	{
 		static void Main(string[] args)
 		{
+			Console.WriteLine("Enter latitude:\t");
+			string latitude = Console.ReadLine()?? throw new FormatException();
+
+            Console.WriteLine("Enter longitude:\t");
+            string longitude = Console.ReadLine() ?? throw new FormatException();
+
+            var rezalt = SBas_homework_API.GetWeather(latitude, longitude).Result;
```

Comment:
`rezalt` ??

.Result is blocking operation use await
to use await mark Main as async, and Task instead of void